sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Models/ContentProperties.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Azure.Core;
+
+namespace Azure.Containers.ContainerRegistry
+{
+    /// <summary>
+    /// </summary>
```

Comment:
Did you intend these to all remain blank for now?

Reply:
I do.  I've filed https://github.com/azure/azure-sdk-for-net/issues/19571 to track this.